tests/ttnn/integration_tests/unet/test_ttnn_shallow_unet.py

```diff
@@ -503,88 +503,92 @@ def forward(self, x):
-    args = ap.parse_args()
-
-    device_id = 0
-    device = ttnn.open_device(device_id=device_id)
-
-    torch.manual_seed(0)
-
-    torch_model = UNet()
-    for layer in torch_model.children():
-        print(layer)
-
-    new_state_dict = {}
-    for name, parameter in torch_model.state_dict().items():
-        if isinstance(parameter, torch.FloatTensor):
-            new_state_dict[name] = parameter + 100.0
-
-    torch_model.load_state_dict(new_state_dict)
-
-    torch_input_tensor = torch.randn(2, 3, 1056, 160)  # Batch size of 2, 3 channels (RGB), 1056x160 input
-    torch_output_tensor = torch_model(torch_input_tensor)
-
-    reader_patterns_cache = {}
-    parameters = preprocess_model(
-        initialize_model=lambda: torch_model,
-        run_model=lambda model: model(torch_input_tensor),
-        custom_preprocessor=create_custom_preprocessor(device),
-        reader_patterns_cache=reader_patterns_cache,
-        device=device,
-    )
-
-    ttnn_model = ttnn_shallow_unet.UNet(parameters)
-
-    #
-    # Tensor Preprocessing
-    #
-    input_shape = torch_input_tensor.shape
-    input_tensor = torch.permute(torch_input_tensor, (0, 2, 3, 1))
-
-    # Pad to 16 if grayskull run and 32 for wormhole
-    input_tensor = input_tensor.reshape(
-        input_tensor.shape[0], 1, input_tensor.shape[1] * input_tensor.shape[2], input_tensor.shape[3]
-    )
-    pad = 32 if device.arch() == ttl.device.Arch.WORMHOLE_B0 else 16
-    input_tensor = torch.nn.functional.pad(input_tensor, (0, pad - input_tensor.shape[-1]))
-    input_tensor = ttnn.from_torch(input_tensor, dtype=ttnn.bfloat16)
-
-    warmup = 1
-    start = None
-    for i in range(args.loop + warmup):
-        if i == warmup:
-            start = time.perf_counter()
-        profiler.tracy_frame()
-        output_tensor = ttnn_model(device, input_tensor)
-    if start is not None:
-        stop = time.perf_counter()
-        total_time = stop - start
-        batch = input_shape[0]
-        total_frame_count = batch * args.loop
-        print(f"Elapsed host time (sec): {total_time}")
-        print(f"Frames processed: {total_frame_count}")
-        print(f"Host perf (fps): {total_frame_count / total_time}")
-
-    #
-    # Tensor Postprocessing
-    #
-    output_tensor = ttnn.to_torch(output_tensor)
-    # unpad to 3
-    output_tensor = output_tensor[:, :, :, :3]
-    output_tensor = output_tensor.reshape(input_shape[0], input_shape[2], input_shape[3], input_shape[1])
-    output_tensor = torch.permute(output_tensor, (0, 3, 1, 2))
-    output_tensor = output_tensor.to(torch_input_tensor.dtype)
-
-    output_tensor = output_tensor[:, 0, :, :]
-    output_tensor = torch.reshape(
-        output_tensor, (output_tensor.shape[0], 1, output_tensor.shape[1], output_tensor.shape[2])
-    )
-    # todo: taps - Disable assert with pcc as pcc is really bad
-    # assert_with_pcc(torch_output_tensor, output_tensor, pcc=0.9999)
-    ttnn.close_device(device)
+    with torch.no_grad():
```

Comment:
I see that this test isn't under a `pytest`.. Does this test actually get run on CI? If not, we should add this so that this gets coverage.

Reply:
fixed